python/.flake8

```diff
@@ -4,4 +4,5 @@ ignore = E203,E501,W503
 max-line-length = 100
 max-complexity = 10
 rst-roles = class,const,func,meth,mod,ref
-rst-directives = deprecated
\ No newline at end of file
+rst-directives = deprecated
+exclude = pysrc/kaskada/timestream/__init__.py
```

Comment:
We shouldn't need to exclude `__init__.py` -- what was `flake8` complaining about?

Reply:
it doesn't like that `Arg, LiteralValue, record, Literal` are imported into `__init__.py` but not used there.

I needed to import them there to resolve the circular reference issues.  basically need all of them + Timestream to exist before the remainder of things.

Reply:
fixed this, removed the flake8 exclude.